docs/basic_construction.md

```diff
@@ -0,0 +1,125 @@
+# Telescope - Basic Construction
+**ALBA** provides a method for a prover to convince a verifier that the prover knows at more than $n_f$ elements satisfying a predicate $R$, while only presenting a small subset of elements as proof. 
+This construction uses a recursive filtering process, guided by random oracles, to produce succinct and efficient proofs.
+
+## Overview
+- The prover builds a proof by creating tuples (sequences) of elements from $S_p$, the set of elements. 
+- The process begins with roots as starting points and expands them step-by-step. 
+- At each step, only tuples that satisfy specific conditions, as determined by a random oracle, are kept for further extension.
+- The random oracle acts as a filter to ensure that the prover selects only a small subset of elements that is highly unlikely to satisfy the conditions by chance alone.
```

Comment:
I am not sure about what you want to mean here?

Reply:
@curiecrypt I am not sure you resolved this.

What are the conditions if not the oracle check? Why would the RO be a filter then?